src/main/kotlin/net/badgersmc/trivia/infrastructure/di/ServiceModule.kt

```
@@ -0,0 +1,141 @@
+    }
+
+    val triviaService: TriviaService by lazy {
+        TriviaService(plugin, config, questionFetcher, statsRepository, nexusScheduler)
+    }
+
+    val chatListener: ChatListener by lazy {
+        ChatListener(triviaService, lang)
+    }
+
+    @Suppress("UNCHECKED_CAST")
+    private fun loadConfig(): TriviaConfig {
```

detekt (1.23.8)

[warning] The function loadConfig appears to be too complex based on Cyclomatic Complexity (complexity: 35). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)